chef/data_bags/secrets/production.json

```diff
@@ -12,7 +12,13 @@
     "version": 1,
     "cipher": "aes-256-cbc"
   },
-  "mysql_password": {
+  "mysql_prod_password": {
+    "encrypted_data": "swX/3e9ap30MOHpSP0Ax4gfNgKVfP3kI8Aw/eQnsJd94OYrRJVl8XRrTGxHq\n0pYdMCM13dcJlLO/fsmCYARX7Q==\n",
+    "iv": "RQufC1hLQBb6XwhTlD0TfQ==\n",
+    "version": 1,
+    "cipher": "aes-256-cbc"
+  },
+  "mysql_staging_password": {
```

Comment:
Is this intentional? Looks like we still only use `mysql_password` in `chef/site-cookbooks/wca/recipes/default.rb`.

Reply:
Oops! Thank you, that was definitely not intentional.